content/blog/2025/good-citizen/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "The difference between upstream-led vs. stakeholder-led open source contributions"
```

Comment:
I would like to spend some time brainstorming different options here. I'm not a big fan of this one, although I agree that the earlier 'strategic vs tacitical' is not appropriate either.

Reply:
For what it's worth I'm not attached to this language at all. Happy to use something else. The reason I moved away from the previous language was (so you know my rationale in case it's helpful for next iteration):

- it wasn't precise enough (and I don't want people to fill in too many missing gaps)
- All of our work needs to be strategic at the end of the day, and in our case getting our product work accomplished through open source development is definitely a strategic choice

Reply:
100% i never liked the previous ones as well for both the reasons you listed, plus the military / violent feel of 'tactical' in my mouth